src/components/TableModule/TableModule.tsx

```diff
@@ -316,14 +316,16 @@ export const TableModule = React.memo(
       }, [headings, sortState.sortDirection, sortState.sortKey]);
 
       React.useEffect(() => {
-        const allStickyCells = Array.from(
-          document.querySelectorAll('.sticky-cell-hook')
-        );
-        allStickyCells.forEach((cell, index) => {
-          if ((index + 1) % stickyCols.length === 0) {
-            cell?.classList.add(classes.isStickyLast);
-          }
-        });
+        if (stickyCols.length > 0) {
```

Comment:
nit: reduce nesting by bailing early in the else case (`if (stickyCols.length === 0) { return; }`)